src/WebJobs.Script/Rpc/RpcServerInitializationService.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Abstractions;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Azure.WebJobs.Script.Rpc
+{
+    public class RpcServerInitializationService : IHostedService
```

Comment:
@brettsam / @fabiocav  - this service needs to be initialized before JobHost starts. Is something like this https://www.nuget.org/packages/AspNetCore.AsyncInitialization/ better than IHostedService?